chapter_introduction/index_vn.md

```diff
@@ -446,13 +446,13 @@ that will follow us around, no matter what kind of ML problem we take on:
 
 Trong ví dụ về *từ đánh thức*, chúng tôi đã mô tả một bộ dữ liệu
 bao gồm các đoạn âm thanh và các nhãn nhị phân,
-giúp các bạn hiểu một cách chung chung về cách chúng ta có thể *huấn luyện*
+giúp các bạn hiểu một cách chung chung về cách *huấn luyện*
```

Comment:
Mấy phần đầu mình dịch còn chưa chia mỗi câu một dòng. Giờ revise thì sửa luôn.

Nếu bạn muốn tránh diff to quá thì làm một PR riêng chỉ để sửa riêng mỗi dòng một câu. Mình sẽ approve ngay.

Reply:
#305